### ISSUE TITLE
Block spurious message: generated new fontManager

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#6202]
This comes out with every cli operation:

`matplotlib/font_manager.py:1547:_load_fontmanager INFO:matplotlib.font_manager:generated new fontManager`

Figure out how to block it.

**Comment by e-carlin** [comment]
Agreed. I thought I had opened a similar issue but can't seem to find it. All spurious message like this should be blocked when running a cli. They prevent piping to other commands.
